### ISSUE TITLE
Bug: Plugin throws error when parsing object shorthand notation

### ISSUE THREAD

**Issue by lephuongbg** [prettier/plugin-pug#147]
## Info

| Tool     | Version       |
| -------- | ------------- |
| Plugin   | v1.10.0        |
| Prettier | v2.1.2        |
| Node     | v14.8.0        |
| OS       | linux |

## Input

```pug
//- template.vue
span {{$t(expanded ? "panel_title_without_title" : "panel_title", {index: index + 1, title})}}
```

## Output or Error

```bash
[PugPrinter:formatText]:  SyntaxError: Missing expected :
    at i (/var/app/web/node_modules/prettier/parser-angular.js:46:16461)
    at e.parseNgInterpolation (/var/app/web/node_modules/prettier/parser-angular.js:46:17640)
    at s (/var/app/web/node_modules/prettier/parser-angular.js:46:28364)
    at Object.e.parseInterpolation (/var/app/web/node_modules/prettier/parser-angular.js:46:28664)
    at /var/app/web/node_modules/prettier/parser-angular.js:46:29282
    at Object.parse (/var/app/web/node_modules/prettier/parser-angular.js:46:28970)
    at Object.parse (/var/app/web/node_modules/prettier/index.js:13575:19)
    at coreFormat (/var/app/web/node_modules/prettier/index.js:14851:25)
    at format (/var/app/web/node_modules/prettier/index.js:15073:14)
    at formatWithCursor (/var/app/web/node_modules/prettier/index.js:15090:12)
```

## Expected Output

```pug
//- template.vue
span {{$t(expanded ? "panel_title_without_title" : "panel_title", {index: index + 1, title})}}
```

## Additional Context
The output seems to be fine (as may be it was kept as-is without any changes, not really sure)

**Comment by Shinigami92** [comment]
I'm sorry to inform you that I will probably not have time until the weekend to keep working on it

**Comment by Shinigami92** [comment]
I have shortened the warning like some others
This has now been released `1.10.1`
Thank you for using my plugin and for being part of the pug community 🤗
